content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,441 @@
+---
+reviewers:
+- fgrzadkowski
+- jszczepkowski
+- directxman12
```

Comment:
@ecojuntak remember to remove reviewers from the final document unless they speak bahasa Indonesia well.

Reply:
Sure